src/components/FlawHistory.vue

```diff
@@ -20,19 +39,20 @@ function isDateField(field: string) {
 
 <template>
   <LabelCollapsible
-    v-if="flaw.history && flaw.history.length > 0"
+    v-if="history && history.length > 0"
     class="my-2"
     :isExpanded="historyExpanded"
+    :isExpandable="history.length > 0"
```

Comment:
I think we don't need this, since the top `v-if` will prevent this component from rendering when the `length` is 0
```suggestion

```